deploy/template/authConfigmap.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: kubecube-auth-config
+  namespace: kubecube-system
+data:
+  github.enabled: enabled
+  github.clientId:
```

Comment:
What if add another way to oAuth? How do we organize the structure to make this look clearer？